lib/exe/css.php

```diff
@@ -103,8 +103,8 @@ function css_out(){
 
     // check cache age & handle conditional request
     // This may exit if a cache can be used
-    http_cached($cache->cache,
-                $cache->useCache(array('files' => $cache_files)));
+    $cache_ok = $cache->useCache(array('files' => $cache_files));
+    http_cached($cache->cache, $cache_ok);
```

Comment:
I'm not sure I understand this change. `$cache->useCache()` will be executed before `http_cached()` either way. Or am I missing something here?

However, I find your version to be more readable  🙂